spacy/lang/nl/lemmatizer/_adverbs_irreg.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf8
+from __future__ import unicode_literals
+
+
+ADVERBS_IRREG = {
+    "'ns": ('eens',),
+    "'s": ('eens',),
+    'acht': ('halfacht',),
```

Comment:
I'm puzzled by `acht` -> `halfacht`

Reply:
That is indeed puzzling. I would suggest manually removing it. I'll investigate how we can avoid the same mistake in future updates.